src/components/shoppingList/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import EditIcon from '@material-ui/icons/Edit';
+import DeleteIcon from '@material-ui/icons/Delete';
+import { ShoppingProps } from "../../utils/interfaces";
+import './styles.css'
+
+const ShoppingList = ({shoppingList}: ShoppingProps): JSX.Element => {
+    return (
+        <div className="shopping-list-container">
+            {shoppingList.map((purchase, index) => {
+                const purchaseNumber = index+1
```

Comment:
Don't create a separate variable for this. Just put index + 1 directly where you need it